homeassistant/components/xiaomi_aqara/binary_sensor.py

```diff
@@ -340,7 +334,7 @@ async def async_added_to_hass(self) -> None:
 
     def parse_data(self, data, raw_data):
         """Parse data sent by gateway."""
-        self._should_poll = False
+        self._attr_should_poll = False
         if NO_CLOSE in data:  # handle push from the hub
```

Comment:
Wasnt there some rule saying we should use the property instead of the attribute even for internal access?

Reply:
It's a write. Not a read.

Reply:
Sorry to early in the morning :)